### PR TITLE
fix: copy operation in IE11

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* fixed copy operation in IE11.
  * capture default copy event and overwrite the event operation

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/editor/src/markdown/mdEditor.ts [hunk 1]
```diff
@@ -195,6 +196,29 @@ export default class MdEditor extends EditorBase {
...
+
+    const text = this.getChanged(selection.content());
+
+    if (ev.clipboardData) {
+      ev.clipboardData.setData('text/plain', text);
+    } else {
+      (window as WindowWithClipboard).clipboardData!.setData('Text', text);
+    }
+
+    if (type === 'cut') {
```
Comment (jung-han) [inline apps/editor/src/markdown/mdEditor.ts:216]: type을 상수로 관리하면 좋을 것 같습니다 ㅎㅎ

**Review by jung-han — APPROVED** [review]
완료입니다!

**Review by jungeun-cho — APPROVED** [review]
이견없습니다.

**Pull request merged**